dist/bin/upgrade-grist

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+docker compose pull
+
+if systemctl is-active --quiet grist.service; then
+  sudo systemctl restart grist.service
```

Comment:
Does doing `down` and `up` with compose refresh the `latest` image? As I thought you had to do a `pull` for that. The upgrade script seems to only restart the service via `down` then `up`, if I haven't missed anything?

Reply:
The pull is on line 5!

Reply:
:facepalm: